modules/installation-configuration-parameters.adoc

```diff
@@ -767,10 +767,16 @@ host must trust the certificate.
 |A map of keys and values that the installation program adds as tags to all resources that it creates.
 |Any valid YAML map, such as key value pairs in the `<key>: <value>` format. For more information about AWS tags, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources] in the AWS documentation.
```

Comment:
Shouldn't this cell require an `a` modifier? https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/#a-operator

Reply:
I'll raise it as a separate issue

Reply:
Yes thank you :)